src/Main.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react'
+import Menu from './components/Menu/Menu';
+import { getAllGoods } from './request';
+
+export class Main extends Component {
```

Comment:
Do not see any reason to have it like class component.

Reply:
not today